esx_service/vmci/connection_types.h

```diff
@@ -29,6 +29,13 @@
 #define START_NON_PRIVILEGED_PORT 1024
 
 /*
+ * Check and set errno helper
+ * Useful when send/recv gets us less than we wanted, abd we want to set errno
```

Comment:
yes, esx_service returns success on create() request for already existing disk .